ResearchKit/Charts/ORKGraphChartView.h

```diff
@@ -0,0 +1,326 @@
+ the largest value of the `maximumValue` property belonging to the `ORKRangedPoint`
+ instances returned by the `ORKGraphChartViewDataSource` protocol's
+ `graphChartView:plot:valueForPointAtIndex:` method.
+*/
+@property (nonatomic, readonly) CGFloat maximumValue;
+
+/**
+ A Boolean value indicating whether the graph view should draw horizontal refrerence lines.
+
+ The default value of this property is NO.
+ */
+@property (nonatomic) BOOL showsHorizontalReferenceLines;
+
+/**
+ A Boolean value indicating whether the graph view should draw vertical refrerence lines.
+
+ The default value of this property is NO.
+*/
+@property (nonatomic) BOOL showsVerticalReferenceLines;
+
+/**
+ The delegate will be notified of pan gesture events occuring within the bounds of the graphChartView.
+
+ See the `ORKGraphChartViewDelegate` protocol.
+*/
+@property (nonatomic, weak, nullable) id <ORKGraphChartViewDelegate> delegate;
+
+/**
+ The dataSource is responsible for providing the data required to populate the graphChartView.
+
+ See the `ORKGraphChartViewDataSource` protocol.
+*/
+@property (nonatomic, weak) id <ORKGraphChartViewDataSource> dataSource;
+
+/**
+ The color of the axes drawn by the graphChartView.
+ 
+ The default value for this property is a very light gray color.
+*/
+@property (nonatomic, strong) UIColor *axisColor;
+
+/**
+ The color of the axes titles.
+ 
+ The default value for this property is a light gray color.
+*/
+@property (nonatomic, strong) UIColor *axisTitleColor;
+
+/**
+ The color of the reference lines.
+ 
+ The default value for this property is a light gray color.
+*/
+@property (nonatomic, strong) UIColor *referenceLineColor;
+
+/**
+ The background color of the thumb on the scrubber line.
+ 
+ The default value for this property is a white color.
+*/
+@property (nonatomic, strong) UIColor *scrubberThumbColor;
+
+/**
+ The color of the scrubber line.
+ 
+ The default value for this property is a gray color.
+*/
+@property (nonatomic, strong) UIColor *scrubberLineColor;
+
+/**
+ The string that will be displayed if no data points are provided by the `dataSource`.
+ 
+ The default value for this property is an appropriate message string.
+*/
+@property (nonatomic, copy, nullable) NSString *noDataText;
+
+/**
+ An image to be optionally displayed in place of the maximum value label on the y-axis.
+ 
+ The default value for this property is nil.
+*/
+@property (nonatomic, strong, nullable) UIImage *maximumValueImage;
+
+/**
+ An image to be optionally displayed in place of the minimum value label on the y-axis.
+ 
+ The default value for this property is nil.
+*/
+@property (nonatomic, strong, nullable) UIImage *minimumValueImage;
+
+/**
+ The gesture recogniser that is used for scrubbint by the graph view.
+ 
+ This object is instatiated and added to the view on initialisation.
+ */
+@property (nonatomic, strong, readonly) UIPanGestureRecognizer *panGestureRecognizer;
```

Comment:
I guess that it could be useful if the user wants to add a graph in a custom view hierarchy that has other conflicting gesture recognizers, and wants to prioritize them. (same as `UIScrollView`'s `panGestureRecognizer` and pinchGestureRecognizer`).